aviary/models/propellers/PropFan.prop

```diff
@@ -1,4 +1,4 @@
-# created 06/24/24 at 17:13 by xjiang
+# created 06/24/24 at 17:13
 # GASP propeller map converted from PropFan.map
 # Propfan format - CT = f(Mach, Adv Ratio & CP)
 # mach_type = mach
```

Comment:
```suggestion
mach_type = mach
```

Reply:
I hesitated to do this change because it will require changes in `read_data_file()` of `aviary.utils.csv_data_file`. Do you think I should do it?

Reply:
would using `mach_type,mach` avoid that problem?

Reply:
The same.